anxcloud/resource_virtual_server_test.go

```diff
@@ -136,39 +136,6 @@ func TestAccAnxCloudVirtualServer(t *testing.T) {
 				ImportStateVerify:       true,
 				ImportStateVerifyIgnore: []string{"critical_operation_confirmed", "enter_bios_setup", "force_restart_if_needed", "hostname", "password", "template", "template_type", "network"},
 			},
-			{
-				Config: testAccConfigAnxCloudVirtualServer(resourceName, templateName, &vmAddTag, "newTag"),
-				Check: resource.ComposeTestCheckFunc(
-					testAccCheckAnxCloudVirtualServerExists(resourcePath, &vmAddTag),
-					resource.TestCheckResourceAttr(resourcePath, "tags.0", "newTag"),
-				),
-			},
```

Comment:
this checks if we can tag the VM as expected, right?

Reply:
Yes this test is to confirm the tagging ability. It was not sufficient. I've reworked v-server tag operation testing in a new commit.

Reply:
tagging should be sufficiently tested with the new generic way to do that, right?

Reply:
Yes. Tagging should work the same for every resource with the new `middleware` approach. That's why I kept the old tagging tests specifically for VMs. But I found an easy way to add the extended testing workflow to the existing vserver test so that we don't need to provision another vm.